client/app/iplb/server/translations/Messages_fr_FR.xml

```diff
@@ -10,6 +10,7 @@
    <translation id="iplb_server_preview_cookie" qtlid="223736">Cookie</translation>
    <translation id="iplb_server_preview_chain" qtlid="19819">Chaine de certification</translation>
    <translation id="iplb_server_preview_proxyProtocolVersion" qtlid="438259">Version du protocole de PROXY</translation>
+   <translation id="iplb_server_preview_proxyProtocolVersion_placeholder">Sélectionnez...</translation>
```

Comment:
does it exist in common?  If so then use it.. else add it

Reply:
I just checked it. It doesn't